packages/tools/devtools/devtools-view/src/components/Menu.tsx

```diff
@@ -197,12 +202,23 @@ function RefreshButton(): React.ReactElement {
  */
 export function MenuSection(props: MenuSectionProps): React.ReactElement {
 	const { header, icon, children, onHeaderClick } = props;
+	const focusable = props.focusable === undefined ? false : true;
 
 	const styles = useMenuSectionStyles();
+	const handleKeyDown = (event: React.KeyboardEvent<HTMLDivElement>): void => {
+		if ((event.key === "Enter" || event.key === " ") && onHeaderClick) {
```

Comment:
Nit: Is there a namespace or enum we can use for these values rather than string literals? E.g. `event.key === Keys.Space`? If so, I think that would be preferrable. But this is fine if there is no such option.